src/BlazorUI/Bit.BlazorUI/Components/Inputs/OtpInput/BitOtpInput.razor.cs

```diff
@@ -273,46 +273,46 @@ private async Task NavigateInput(string code, string key, int index)
             await Task.Delay(1);
             await _inputRefs[previousIndex].FocusAsync();
         }
-        else if (code is "ArrowLeft")
+        else if (code is "ArrowLeft" && Vertical is false)
```

Comment:
plz improve this method as we talked.

Reply:
We discussed that we do it in Dir PR not this one :)